website/docs/reference/resource-configs/alias.md

```diff
@@ -4,10 +4,47 @@ description: "Read this guide to understand the alias configuration in dbt."
 datatype: string
 ---
 
-:::caution Heads up!
-This is a work in progress document. While this configuration applies to multiple resource types, the documentation has only been written for seeds.
 
-:::
+<Tabs>
+<TabItem value="model" label="Model">
+
+Specify a custom alias for a model in your `dbt_project.yml` file. 
+
+For example, if you have a model that calculates `sales_metric`s and want to give it a more user-friendly alias, you can alias it like this:
+
+<File name='dbt_project.yml'>
+
+```yml
+models:
+  your_project:
+    sales_metrics:
+      +alias: sales_dashboard
+```
+</File>
+
+This would return `analytics.finance.sales_dashboard` in the database, instead of the default `analytics.finance.sales_metrics`.
+</TabItem>
```

Comment:
or is it `analytics.finance.unique_order_id_test`? [this](https://docs.getdbt.com/reference/resource-properties/tests#custom-test-name) makes me think it's the _ one?